.docs/user-guide/troubleshooting.md

```diff
@@ -0,0 +1,241 @@
+# Troubleshooting
+
+It's not doing what I expected...
+
+---
+
+## Configuration Validation
+REX-Ray configuration will usually be set in the yaml file located at `/etc/rexray/config.yml`
+
+If you are having issues, or anytime you edit your configuration you may wish to validate the result by cut and paste to an online validator such as this one: [http://yaml-online-parser.appspot.com/](http://yaml-online-parser.appspot.com/)
+
+If REX-Ray is running as a service, the service needs to be restarted to pick up the change.
+
+`sudo rexray service restart`
+
+The configuration file is the conventional method to set configuration, but this can be supplemented or replaced by environment variables and command line option flags. For example, when running a centralized REX-Ray controller service in a Docker container, you would likely use an environment variables configuration. 
+
+If configuration is set in in multiple locations, the precedence order is:
+
+Command line overrides -> environment overrides -> config file
+
+The command `rexray env` can be used to dump the current _effective_ configuration - in an environment variable form, regardless of whether environment variables were used to set the configuration.
+
+### Ultra simplified configuration 
+A minimal complexity configuration can be used as an experiment to eliminate issues external to configuration.
+
+1. Shut down the REX-Ray service: `sudo rexray service stop`
+2. Save your original config: `sudo mv /etc/rexray/config.yml /etc/rexray/config.$(date +%Y-%m-%d-%T).yml`
+3. Create a new minimal config `vi \etx\rexray\config.yml`
+
+This is a minimal configuration for **AWS EBS** volumes. Other storage drivers are similar and will mostly vary along the lines of authorization credentials, and possibly storage API address identification.
+
+```
+rexray:
+  loglevel:warn
```

Comment:
Hi @cantbewong,

The default log level is `warn`, so duplicating here is not necessary.